### PR TITLE
Data Augmentation experiment

### PR DESCRIPTION
**Pull request by adamantivm**
Trying various things with not much better results using an experiment file.

Updated the experiment file to log to MLFlow, including the inference metrics.

### CODE REVIEW

File: experiments/2024_12_17_julian_amped_augmentation.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "normalize_output": False,
+    "equalization": True,
+    # Training parameters
+    "n_epochs": 1000,
+    "learning_rate": 1e-4,
+    "random_rotation_degrees": 10,
+    "crop_box_jitter": [5, 20],
+    "color_jitter": 0.5,
+    # Model parameters
+    "dropout_p": 0.1,
```
Comment (alejandromarcu) [inline experiments/2024_12_17_julian_amped_augmentation.py:38]: You recently said that dropout made it worse 🤔

File: experiments/2024_12_17_julian_amped_augmentation.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    test_df = run_dataset_inference(
+        model,
+        test_dataset_path,
+        annotations_file,
+        **preprocessing_args,
+        use_water_line=train_water_line,
+    )
+
+    plot_inference_results(test_df, train_df, plot_filename="inference_results.png")
+    mlflow.log_artifact("inference_results.png")
```
Comment (alejandromarcu) [inline experiments/2024_12_17_julian_amped_augmentation.py:78]: nice!
